bin/src/hickory-dns.rs

```diff
@@ -243,6 +248,27 @@ fn run() -> Result<(), String> {
         info!("Prometheus metrics are disabled");
     }
 
+    #[cfg(feature = "metrics")]
+    let process_metrics_collector = {
+        // setup process metrics (cpu, memory, ...) collection
+        let collector = Collector::default();
+        collector.describe(); // add metric descriptions
+
+        runtime.spawn(async move {
+            loop {
+                tokio::time::sleep(Duration::from_millis(250)).await;
```

Comment:
Nit: import `sleep()` directly. Why 250ms?

Reply:
The 250ms had been freely chosen, ideally this value should be smaller than the metrics endpoint scrape interval. Typical default intervals are 10s to 1m, but I've already encountered systems with shorter intervals for scraping.

250ms seemed to be a reasonable value, ensuring reasonable granularity for systems with short scrape intervals, but likely it could be extended to a higher value.

Reply:
Moved the fully qualified sleep method call to an import.
Increased the interval to 1s.